scripts/src/templates/dataset-mixed-content-modal.js

```diff
@@ -0,0 +1,21 @@
+export default (resourceUrl, originalDatasetUrl) => {
+    return `<div class="modal fade" tabindex="-1" role="dialog" id="mixed-content-warning-modal">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title">Mixed content warning</h4>
+      </div> 
+      <div class="modal-body">
+        <p>This resource link is only served over HTTP. As such, we can not link to it as most modern browsers do not allow direct links from HTTPS secured sites to HTTP sites for security reasons.</p>
+        <p>To access the resource, please copy the following URL and paste it into the address bar of your browser:</p>
+        <pre>${resourceUrl}</pre>
+        <p>Alternatively, you can access the resource via the <a href="${originalDatasetUrl}" target="_blank">original dataset link</a>.</p>
```

Comment:
Original dataset url not quite working.
<img width="861" alt="Screenshot 2023-02-22 at 23 07 34" src="https://user-images.githubusercontent.com/47697803/220784487-86d0a225-5d54-457a-a534-b87973986134.png">

Reply:
Helps if I grab the URL correctly 🤦‍♂️ Fixed now!